spec/models/game_spec.rb

```diff
@@ -43,4 +43,9 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  it "should have correct number of pieces" do
+      game = Game.create
+      game.pieces.count==32
```

Comment:
I see. But I tried to use 'expect(game.pieces.count).to eq(32)' first. And it got 0 not 32.